http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonJacksonPayloadIT.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Dependency;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class JsonJacksonPayloadIT extends JsonPayloadIT {
+
+    private static final String REST_JACKSON = "rest-jackson";
+
+    @QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class, Person.class,
+            PersonResource.class }, properties = "oidcdisable.properties", dependencies = @Dependency(artifactId = "quarkus-"
+                    + REST_JACKSON))
+    static RestService app = new RestService();
+
+    @Override
+    protected RestService getApp() {
+        return app;
+    }
+
+    @Test
+    public void shouldPickJackson() {
+        assertTrue(app.logs().forQuarkus().installedFeatures().contains(REST_JACKSON));
```

Comment:
Isn't `shouldPickJackson` testing that `rest-jackson` feature is present? If you compare logs of `JsonJacksonPayloadIT`: 
```
2024-07-24T08:07:28.0863296Z 08:07:28,083 INFO  [app] 08:07:26,210 Installed features: [cdi, keycloak-authorization, micrometer, oidc, reactive-routes, rest, rest-client, rest-jackson, rest-jaxb, rest-jsonb, security, smallrye-context-propagation, smallrye-health, smallrye-openapi, vertx]
```

with `JsonBPayloadIT` logs:

```
2024-07-24T08:07:19.4630743Z 08:07:19,461 INFO  [app] 08:07:17,767 Installed features: [cdi, keycloak-authorization, micrometer, oidc, reactive-routes, rest, rest-client, rest-jaxb, rest-jsonb, security, smallrye-context-propagation, smallrye-health, smallrye-openapi, vertx]
```

but both of these tests have `rest-jaxb` and `rest-jsonb`features present as well.
 
Based on what can you determine that these 2 tests are using different serializers? Let alone that you are not actually using these extensions to serialize/deserialize payload, but that's a different story.

Reply:
When reviewing this I didn't realized that `@Dependency(artifactId = "quarkus-" + REST_JACKSON)` add it but the extension defined in pom stay there as well (probably that I was looking at [GreetingResourceIT](https://github.com/quarkus-qe/quarkus-test-framework/blob/main/examples/blocking-reactive-model/src/test/java/io/quarkus/qe/GreetingResourceIT.java) and didn't look at the pom there).

 > Based on what can you determine that these 2 tests are using different serializers?
 Now I can say we can't say for sure. As I described above I was thinking that there is only that one extension.

Reply:
I think it still can be sure that Quarkus REST Jackson serializers have a higher priority, but I simply don't know it and we don't assert it, therefore we need a way to assert it. Thank you